README.md

```diff
@@ -103,6 +103,14 @@ Options:
   -h, --help                display help for command
 ```
 
+### npm executable
+
+By default, this tool will use global `npm`. This can  be changed by setting environmental variable `npm_execpath` to path of `npm-cli.js` executable, for example:
```

Comment:
the global - or the currently running `npm` executable.

background: 
as soon as `npm` spawns a sub-process - for `npx`, `npm exec`, `npm script` - it will set this variable "npm_execpath" to the proper value.
and node internally also honors this variable.